src/registrar/models/domain.py

```diff
@@ -228,6 +228,44 @@ def password(self) -> str:
         """
         raise NotImplementedError()
 
+    @Cache
+    def format_nameservers(self):
+        """
```

Comment:
So this does violate the MVT set up, your model ideally shouldn't dictate any logic with formatting. The model should just handle CRUD operations on the data itself, while View is for data processing pre/post retrieval as well as http/rendering, and then the template should be your format for your page. Ideally this should all go in the template, however as it does involve data manipulation for the sole purpose of display this sounds like you can just cart this over to the view instead.

Reply:
Yes 100%!! This was totally my bad I smooth brained and put it the file and didn't even think about logic vs view side. Will remedy this! Thank you for catching this, I totally borked on this 😵‍💫